packages/reusable-blocks/src/components/reusable-blocks-menu-items/reusable-block-convert-button.js

```diff
@@ -51,8 +51,16 @@ export default function ReusableBlockConvertButton( {
 	const canConvert = useSelect(
 		( select ) => {
 			const { canUser } = select( coreStore );
-			const { getBlocksByClientId, canInsertBlockType } =
-				select( blockEditorStore );
+			const {
+				getBlocksByClientId,
+				canInsertBlockType,
+				getBlockRootClientId,
+			} = select( blockEditorStore );
+
+			const rootId =
+				rootClientId || clientIds.length > 0
+					? getBlockRootClientId( clientIds[ 0 ] )
+					: undefined;
```

Comment:
This is wrong, no? It will call `getBlockRootClientId` if `rootClientId` is truthy. I think you need parens around the right hand side of the `||`.

```js
const rootId = rootClientId || ( clientIds.length > 0 ? getBlockRootClientId( clientIds[ 0 ] ) : undefined );
```

Reply:
🤦 should have stuck with the nested ternary I first had, it was easier to grok 😄  It works as expected when properly bracketed and have updated, but I wonder if something like the following is clearer for our future selves and not much more verbose 🤔 :
```js
let rootId;
if ( rootClientId ) {
   rootId = rootClientId;
} else if ( clientIds.length > 0 ) {
   rootId = getBlockRootClientId( clientIds[ 0 ] );
}
```

Reply:
Up to you 😀